node/node_newblock.go

```diff
@@ -127,7 +127,7 @@ func (node *Node) proposeNewBlock() (*types.Block, error) {
 
 	// Prepare cross links
 	var crossLinksToPropose types.CrossLinks
-	if node.NodeConfig.ShardID == 0 {
+	if node.NodeConfig.ShardID == 0 && node.Blockchain().Config().IsStaking(node.Worker.GetCurrentHeader().Epoch()) {
```

Comment:
and this is where we turn off for the leader "until the staking epoch"?
Is the intention to disable crosslink during the pre-staking period?
What is the reason for turning off the crosslink?

cc: @harmony-ek

Reply:
Cross link potentially is causing network trouble.